packages/slate/src/models/node.js

```diff
@@ -1980,10 +1980,7 @@ class Node {
     // If the child is a text node, the `position` refers to the text offset at
     // which to split it.
     if (child.object == 'text') {
-      const befores = child.characters.take(position)
-      const afters = child.characters.skip(position)
-      one = child.set('characters', befores)
-      two = child.set('characters', afters).regenerateKey()
+      ;[one, two] = child.splitText(position)
```

Comment:
Is this `;` necessary?

Reply:
Fixed; Thank you

Reply:
Not fixed; prettier eslint suggest a `;` here.